pandas/tests/test_strings.py

```diff
@@ -1992,6 +1992,19 @@ def test_rsplit(self):
         exp = Series([['a_b', 'c'], ['c_d', 'e'], NA, ['f_g', 'h']])
         tm.assert_series_equal(result, exp)
 
+    def test_split_blank_string(self):
+        # expand blank split GH 20067
+        values = Series([''])
```

Comment:
can you add a ``name=`` to the Series so its clear where the name goes.

Reply:
Not sure I follow exactly what you mean. Are you suggesting adding `name='test'` as an arg to the constructor?

Reply:
yes add a name, the point is that this generates the column name for the frame.